Makefile

```diff
@@ -114,7 +123,7 @@ clean:
 ## Development builds
 ## ------------------
 
-all: format build test error-code
+all: format build test
```

Comment:
Why remove error-code?

Reply:
It is already a dependency of "build"

Reply:
That seems like a mistake and that the approach in this PR is better. But I can send a separate PR for that.